README.md

```diff
@@ -224,6 +225,11 @@ This permission is required to discover protected resources for the AWS/DDoSProt
 "shield:ListProtections"
 ```
 
+This permission is required to get the AWS account alias, used in the the `aws_account_info` metric
```

Comment:
maybe useful to include a link to https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html?

```suggestion
This permission is required to get the [AWS account alias](https://docs.aws.amazon.com/IAM/latest/UserGuide/console_account-alias.html), used in the the `aws_account_info` metric
```

Reply:
did a mix of both